cards/financial-professional-location/component.js

```diff
@@ -6,7 +6,7 @@ class financial_professional_locationCardComponent extends BaseCard['financial-p
   }
 
   onMount() {
-    {{> static/js/interactive-map/location-card/add-card-click-listener}}
+    window.InteractiveMap && new CardListenerAssigner({card: this}).addListenersToCard();
```

Comment:
it may be more telling that someone is using the `InteractiveMap` by the presence of the `InteractiveMap` class in the document. `window.InteractiveMap` may just mean that they accidentally left the bundle on the page.